build/build.go

```diff
@@ -82,6 +82,7 @@ type Options struct {
 	Session      []session.Attachable
 	Linked       bool // Linked marks this target as exclusively linked (not requested by the user).
 	PrintFunc    *PrintFunc
+	MetadataFile string
```

Comment:
As the value itself is not used (afaics) this could be `WithProvenanceResponse bool` or smth

Reply:
At first I used a bool but then I was thinking we could directly handle the metadata file in build package instead of command one so I switched to this type instead. But agree that it seems more explicit to use a bool for this case.